ClickHouse.Client/ADO/ClickHouseConnection.cs

```diff
@@ -277,7 +277,28 @@ public override async Task OpenAsync(CancellationToken cancellationToken)
     /// <param name="isCompressed">indicates whether "Content-Encoding: gzip" header should be added</param>
     /// <param name="token">Cancellation token</param>
     /// <returns>Task-wrapped HttpResponseMessage object</returns>
-    public async Task PostStreamAsync(string sql, Stream data, bool isCompressed, CancellationToken token)
+    public Task PostStreamAsync(string sql, Stream data, bool isCompressed, CancellationToken token)
```

Comment:
This breaks public interface - even though it's experimental. 
While I understand adding extra `await` introduces additional state machine, I feel it's not warranted to break it in this case